distribution/sender/routes/sync-response-processor.xml

```diff
@@ -0,0 +1,66 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sync-response-processor" errorHandlerRef="shutdownErrorHandler">
+        <!-- TODO(luis.pinto) Make the delay configurable -->
+        <from uri="scheduler:sync_response_processor?initialDelay=15000&amp;delay=60000" />
+
+        <log message="Fetching next 1000 sync responses" loggingLevel="DEBUG" />
+
+        <toD uri="jpa:SyncResponse?query=SELECT r FROM SyncResponse r WHERE r.status = 'NEW' ORDER BY r.dateSent, r.id ASC&amp;maximumResults=1000" />
+
+        <choice>
+            <when>
+                <simple>${body.size()} > 0</simple>
+                <log message="Fetched ${body.size()} sender sync responses" />
+                
+                <split parallelProcessing="false">
+                    <simple>${body}</simple>
+                    
+                    <setProperty name="syncResponse">
+                        <simple>${body}</simple>
+                    </setProperty>
+
+                    <log message="Processing sender sync response -> ${exchangeProperty.syncResponse}" />
+                    
+                    <toD uri="jpa:SenderSyncMessage?query=SELECT m FROM SenderSyncMessage m WHERE m.messageUuid='${exchangeProperty.syncResponse.messageUuid}'" />
+                    
+                    <choice>
+                        <when>
+                            <simple>${body.size()} > 0</simple>
+                            <log message="Fetched ${body.size()} Sender sync messages with uuid ${exchangeProperty.syncResponse.messageUuid}" />
+            
+                            <log loggingLevel="DEBUG" message="Sender sync message(s) found -> ${body}" />
+                            
+                            <log message="Removing Sender sync message(s) with uuid ${exchangeProperty.syncResponse.messageUuid}" />
+            
+                            <toD uri="jpa:SenderSyncMessage?query=DELETE FROM SenderSyncMessage WHERE messageUuid = '${exchangeProperty.syncResponse.messageUuid}'" />
+            
+                            <log loggingLevel="DEBUG" message="Successfully removed Sender sync message(s) with uuid ${exchangeProperty.syncResponse.messageUuid}" />
```

Comment:
I would rather have this message logged at info and the one on line 33 at debug

Reply:
Fixed